bin/brunch

```diff
@@ -29,6 +29,7 @@ var exists = function(path) {
 };
 
 var runSeparateWatchProcess = function(cliPath) {
+  process.env.BRUNCH_FORKED_PROCESS = true;
```

Comment:
Have you tested this will work? I think env can be passed to `child_process` call explicitly